backend/associations/serializers/election.py

```diff
@@ -1,83 +1,97 @@
 from rest_framework import serializers
 from rest_framework.exceptions import ValidationError
 
-from associations.models import Association, Election, Choice, Ballot
+from associations.models import Association, Election, Choice, Voter
 from authentication.models import User
+from authentication.serializers.user_short import UserShortSerializer
 
 
-class ChoiceShortSerializer(serializers.ModelSerializer):
+class VoteSerializer(serializers.Serializer):
+    """This serializer is not bound to any model and is used for the /vote/ action."""
+
+    choices = serializers.PrimaryKeyRelatedField(
+        queryset=Choice.objects.all(), many=True, read_only=False
+    )
+
+    def create(self, validated_data):
+        raise NotImplementedError(
+            "This serializer cannot be used for database operations."
+        )
```

Comment:
Why don't you use the same pattern as in `ChoiceReadOnlySerializer` ?

Reply:
The same pattern? What do you mean?